### PR TITLE
fix(ctb): Port All Test Contracts to Triple Slash Natspec Styling

### PR DESCRIPTION
**Pull request by refcell**
**Description**

This PR closes out the port of smart contracts to triple slash natspec styling.

It ports all contracts in the ctb `test` directory to use triple slash natspec
styling from seaport styling.

The `invariant-doc-gen.ts` script is also updated to parse triple slash natspec
styling. Using triple slashes without header and footer prefixes and suffixes
actually serves to simplify the script quite a bit as well.

### CODE REVIEW

**Comment by refcell** [comment]
Current dependencies on/for this PR:
* develop
  * **PR #6401** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/6401" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/ethereum-optimism/optimism/6401?utm_source=stack-comment).

File: packages/contracts-bedrock/invariant-docs/ResourceMetering.md [hunk 1]
```diff
@@ -1,42 +1,2 @@
 # `ResourceMetering` Invariants
 
-## The base fee should increase if the last block used more than the target amount of gas
-**Test:** [`ResourceMetering.t.sol#L180`](../contracts/test/invariants/ResourceMetering.t.sol#L180)
-
-If the last block used more than the target amount of gas (and there were no empty blocks in between), ensure this block's baseFee increased, but not by more than the max amount per block. 
-
-
```
Comment (tynes) [inline packages/contracts-bedrock/invariant-docs/ResourceMetering.md:8]: Where did these go?

Comment (refcell) [reply packages/contracts-bedrock/invariant-docs/ResourceMetering.md:8]: Oh good catch - they really went like this
image

Comment (refcell) [reply packages/contracts-bedrock/invariant-docs/ResourceMetering.md:8]: Found the issue - working on a fix.

Comment (refcell) [reply packages/contracts-bedrock/invariant-docs/ResourceMetering.md:8]: Fixed.

**Review by tynes — APPROVED** [review]

**Comment by tynes** [comment]
Looks like some line length lintings are failing now

**Pull request merged**